bpf/complexity-tests/510/bpf_wireguard/1.txt

```diff
@@ -0,0 +1,37 @@
+-DHAVE_FIB_IFINDEX=1
+-DHAVE_FIB_NEIGH=1
+-DSKIP_DEBUG=1
+-DDISABLE_LOOPBACK_LB=1
+-DENABLE_IPV4=1
+-DENABLE_IPV6=1
+-DENABLE_ROUTING=1
+-DPOLICY_VERDICT_NOTIFY=1
+-DALLOW_ICMP_FRAG_NEEDED=1
+-DENABLE_IDENTITY_MARK=1
+-DMONITOR_AGGREGATION=3
+-DCT_REPORT_FLAGS=0x0002
+-DENABLE_HOST_FIREWALL=1
+-DENABLE_ICMP_RULE=1
+-DENABLE_CUSTOM_CALLS=1
+-DENABLE_SRV6=1
+-DENABLE_MASQUERADE_IPV4=1
+-DENABLE_MASQUERADE_IPV6=1
+-DENABLE_SRC_RANGE_CHECK=1
+-DENABLE_NODEPORT=1
+-DENABLE_NODEPORT_ACCELERATION=1
+-DENABLE_SESSION_AFFINITY=1
+-DENABLE_DSR_ICMP_ERRORS=1
+-DENABLE_DSR=1
+-DENABLE_DSR_HYBRID=1
+-DENABLE_IPV4_FRAGMENTS=1
+-DENABLE_TPROXY=1
+-DENABLE_HOST_ROUTING=1
+-DENABLE_BANDWIDTH_MANAGER=1
+-DETH_HLEN=0
+-DENCAP_IFINDEX=1
+-DTUNNEL_MODE=1
+-DENABLE_EGRESS_GATEWAY=1
+-DENABLE_VTEP=1
+-DENABLE_SCTP=1
+-DENABLE_WIREGUARD=1
+-DIS_BPF_WIREGUARD=1
```

Comment:
Note that IS_BPF_WIREGUARD shouldn't be needed here - it already gets defined by the program itself.

(the other complexity configs don't specify their `IS_BPF_*` value)